### DISCUSSION TITLE
[Auth Middleware] - Not working in subdomain in Adonis 5.0

### DISCUSSION THREAD

**Discussion by BilalBasha** [adonisjs/core#1328] [Help] [answered]
I am having example.com, I am logging in example.com, after logged in my cookies are stored in .example.com, If i access auth routes in example.com is working fine, **but not working in app.example.com**

Can anyone help me out if this

**Comment by hkmsadek** [comment, marked as answer]
This should help 

```
   cookie: {
    httpOnly: true,
    path: '/',
    sameSite: false,
    domain: '.example.com' // add this extra line session.js file
  },

```
